EtchABot.cpp

```diff
@@ -26,15 +26,15 @@ EtchABot::EtchABot(int type, int h1=5, int h2=4, int h3=3, int h4=2,   //Motor p
 	_motorPins[0] = h1;
 	_motorPins[1] = h2;
 	_motorPins[2] = h3;
-	_motorPins[4] = h4;
-	_motorPins[5] = v1;
-	_motorPins[6] = v2;
-	_motorPins[7] = v3;
-	_motorPins[8] = v4;
-	_motorPins[9] = e1;
-	_motorPins[10] = e2;
-	_motorPins[11] = e3;
-	_motorPins[12] = e4;
```

Comment:
everything after the missing [3] was assigned to the next higher pin, resulting in this one out of bounds rather than a motorPins array element assignment.